src/components/countdown/CountdownOutput.vue

```diff
@@ -11,13 +11,19 @@ export default {
   extends: BaseOutput,
   computed: {
     remaining () {
+      const hours = this.settings.hours.toString()
       const minutes = this.settings.minutes.toString().padStart(2, '0')
       const seconds = this.settings.seconds.toString().padStart(2, '0')
-
-      return `${minutes}:${seconds}`
+      if (this.settings.klok === 1) {
+        return this.settings.now.format('H:mm:ss')
+      } else if (this.settings.hours > 0) {
+        return `${hours}:${minutes}:${seconds}`
+      } else {
+        return `${minutes}:${seconds}`
+      }
```

Comment:
Ik zou bovenstaande iets anders schrijven:
```js
if (this.settings.klok === 1) {
  return this.settings.now.format('H:mm:ss') // "Early return", code hieronder is niet relevant indien je de klok gebruikt.
}

const hours = this.settings.hours.toString()
const minutes = this.settings.minutes.toString().padStart(2, '0')
const seconds = this.settings.seconds.toString().padStart(2, '0')

if (this.settings.hours > 0) {
  return `${hours}:${minutes}:${seconds}`
} else {
  return `${minutes}:${seconds}`
}
```

Reply:
omzetting naar string hoeft niet klopt; pas het aan.

Reply:
Het stoppen van de klok gebeurt o.b.v. `this.settings.hours` en niet o.b.v. deze lokale `const hours`. Het klopt idd dat code onder de return niet wordt uitgevoerd, maar dat is volgens mij dus ook in het geval van de klok niet nodig.